components/function-controller/pkg/apis/serverless/v1alpha1/function_defaults.go

```diff
@@ -89,3 +93,9 @@ func (spec *FunctionSpec) defaultResources(ctx context.Context) {
 		spec.Resources.Limits[corev1.ResourceCPU] = newRerource
```

Comment:
I know it is not a line you've changed, just found a typo:
`newRerource` -> `newResource`